lively.morphic/layout.js

```diff
@@ -361,41 +363,7 @@ export class TilingLayout extends Layout {
   attach () {
     this.initializeResizePolicies();
     super.attach();
-    this.measureAfterRender(this.container);
-  }
-
-  retriggerLayoutOfHierarchy () {
-    let curr = this.container;
-    while (
-      curr.owner?._yogaNode &&
-      curr.owner.layout?.name() === 'Tiling' &&
-      curr.isLayoutable &&
-      (curr.clipMode === 'visible' ||
-       curr.layout.hugContentsVertically ||
-       curr.layout.hugContentsHorizontally)
-    ) curr = curr.owner;
-    curr.layout._retriggerLayout = true;
-  }
-
-  ensureLayoutComputed (curr) {
-    const origNode = this.ensureYogaNodeFor(curr);
-    while (curr.owner?._yogaNode && curr.owner.layout?.name() === 'Tiling' && curr.isLayoutable) curr = curr.owner;
-    curr._yogaNode.calculateLayout(curr.width, curr.height);
-    return origNode;
-  }
-
-  ensureYogaNodeFor (aMorph) {
-    return aMorph._yogaNode || (aMorph._yogaNode = Yoga.Node.create(yogaConfig));
-  }
-
-  resetYogaNodeFor (aMorph) {
-    if (!aMorph._yogaNode) return;
-    // clear and free the yoga object
-    aMorph._yogaNode.free();
-    delete aMorph._yogaNode;
-    // ensure that the morph gets rendered again, so that yoga is getting re-initialize
-    aMorph.makeDirty();
-    delete aMorph._lastComputed;
+    this.measureAfterRender(this.container); // no longer needed with yoga??
```

Comment:
Can we either remove the code or the comment?

Reply:
Remove, we will.